package.json

```diff
@@ -53,7 +53,8 @@
       "@types/eslint": "^8.37.0",
       "babel-plugin-module-resolver@5.0.1": "5.0.0",
       "browserslist": "^4.14.0",
-      "graceful-fs": "^4.0.0"
+      "graceful-fs": "^4.0.0",
+      "@embroider/addon-shim": "workspace:*"
```

Comment:
How confident are we that this pattern works in pnpm? Because I tried this locally and still saw packages depending on a copy of addon-shim that came from the NPM registry, not the local one.

Also: I see why we would need this config to test this particular PR. But I don't think we should keep it after this fix is released. It's good that our tests are forced to interact with previously released versions, since that's more realistic.